src/main/java/nl/novi/techiteasycontroller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,39 @@
+package nl.novi.techiteasycontroller.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
```

Comment:
Je kan hier ook een @RequestMapping("/televisons") zetten zodat je dat niet in elke getMapping hoeft te zetten